modules/virt-troubleshooting-vm-import.adoc

```diff
@@ -1,9 +1,9 @@
 // Module included in the following assemblies:
 // * virt/virtual_machines/importing_vms/virt-importing-vmware-vm.adoc
 [id='troubleshooting-vmware-vm-import_{context}']
-= Troubleshooting a VMware virtual machine import
+= Troubleshooting a virtual machine import
 
-If an imported virtual machine's status is `Import error: (VMware)`, you can check the Conversion Pod log for errors:
+If an imported virtual machine's status is `Import error`, you can check the Conversion Pod log for errors:
```

Comment:
According to [here](https://importing_rhv_cnv--ocpdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/importing_vms/virt-importing-rhv-vm.html#troubleshooting-vmware-vm-import_virt-importing-rhv-vm) it seems that it is relevant only for importing a vm from vmware, but there is no notation for it.

The logs of rhv import process should be read from vm-import-controller-xzy pod.

Reply:
@masayag  What error would a RHV VM show?